3.process-cfret-features/extraction_utils.py

```diff
@@ -0,0 +1,98 @@
+"""
+This file holds functions to extract image features from the outputted sqlite file from CellProfiler. These functions are based on the functions from the
+cells.SingleCells class in Pycytominer. This file also hold a function to add single cell counts
+to the single cell count dataframes.
+"""
+
+import pandas as pd
+from sqlalchemy import create_engine
+import numpy as np
+import pathlib
+
+
+def load_sqlite_as_df(
+    sqlite_file: str,
+    image_table_name: str = "Per_Image",
+) -> pd.DataFrame:
+    """
+    load in table with image feature data from sqlite file
+
+    Parameters
+    ----------
+    sqlite_file : str
+        string of path to the sqlite file
+    image_table_name : str
+        string of the name with the image feature data (default = "Per_Image")
+
+    Returns
+    -------
+    pd.DataFrame:
+        dataframe containing image feature data
+    """
+    engine = create_engine(sqlite_file)
+    conn = engine.connect()
+
+    image_query = f"select * from {image_table_name}"
```

Comment:
Consider adding a short comment here to explain what these next two lines do.

Reply:
Done!